rust/relay/server/src/ebpf/linux.rs

```diff
@@ -95,7 +101,7 @@ impl Program {
                             tracing::warn!(%cpu_id, num_lost = %events.lost, "Lost perf events");
                         }
 
-                        tracing::debug!(%cpu_id, num_read = %events.read, "Read perf events from eBPF kernel");
+                        tracing::trace!(%cpu_id, num_read = %events.read, "Read perf events from eBPF kernel");
```

Comment:
This was demoted to trace because it is quite spammy with any level of relayed packets.